Day-II-Calculator/calculator/src/components/ButtonComponents/Button.css

```diff
@@ -0,0 +1,92 @@
+.buttonPanelContainer {
+    display: flex;
+    flex-direction: row;
+    width: 100%;
+    height: 100%;
+}
+
+.buttonPanelNumber{
+    display: flex;
+    flex-direction: column;
+    width: 75%;
+    background: white;
+}
+
+.buttonPanelAction{
+    display: flex;
+    flex-direction: column;
+    width:25%;
+}
+
+.buttonPanelClear{
+    display: flex;
+    flex-direction: column;
+    width:100%;
+    height: 20%;
+    background: white;
+}
+
+.buttonPanelRow {
+    display: flex;
+    flex-direction: row;
+    width:100%;
+    height: 20%;
+    background: white;
+  
+}
+
+.singleButton {
+    display: flex;
+    flex-direction: row;
+    width:100%;
+    background: white;
+}
+
+.singleButton.wide {
+    display: flex;
+    flex-direction: row;
+    width:100%;
+    height: 100%;
```

Comment:
you're repeating a lot of styles that you already get with the .singleButton class